mantle/kola/harness.go

```diff
@@ -872,7 +872,7 @@ func registerTestDir(dir, testprefix string, children []os.FileInfo) error {
 				return errors.Wrapf(err, "reading %s", c.Name())
 			}
 			userdata = conf.Ignition(string(v))
-		} else if isreg && (c.Name() == "config.bu" || c.Name() == "config.fcc") {
+		} else if isreg && (c.Name() == "config.bu") {
```

Comment:
I'd say this is the one piece I'm not sure about. I'm not convinced we should rip out support for `config.fcc` just yet, but just undocument it and also switch any places that are using it to not use it.

@jlebon, @bgilbert, @cgwalters - thoughts?

Reply:
Agree. We can keep it as it is for now although I don't see it getting used after we changed all kola test files from .fcc to .bu.

Reply:
It should be trivial to adapt whatever breaks if so, but agreed we don't really have to be that aggressive either. Maybe we can just print a warning for now so if this shows up somewhere we have a higher chance of noticing?

Reply:
If we think the disruption will be minimal to anything still using `.fcc` then I say let's go for it now. Anything that doesn't require a followup is a win IMO.

Reply:
I generally lean towards warn + sleep.

Reply:
Alright, we can go with something like 
```
if (c.Name() == "config.fcc") {
 return errors.Wrapf(err, "%s not supported anymore", c.Name())
}
```